gosheets.scss

```diff
@@ -0,0 +1 @@
+@warn 'gosheets!!!!';
```

Comment:
Should we still add an empty line at the end of the file?

Reply:
Yep, I will rebase that in that change before I merge.